ddtrace/llmobs/_experiment.py

```diff
@@ -128,8 +141,59 @@ def run(self, jobs: int = 1, raise_errors: bool = False, sample_size: Optional[i
         self._run_evaluators(task_results, raise_errors=raise_errors)
         return
 
+    def _process_record(self, idx_record: Tuple[int, DatasetRecord]) -> Dict[str, Any]:
+        idx, record = idx_record
+        start_ns = time.time_ns()
+        with self._llmobs_instance._experiment(name=self._task.__name__, experiment_id=self._id) as span:
+            span_context = self._llmobs_instance.export_span(span=span)
+            span_id = span_context.get("span_id", "")
+            trace_id = span_context.get("trace_id", "")
+            input_data = record["input_data"]
+            record_id = record.get("record_id", "")
+            expected_output = record["expected_output"]
+            tags = {"dataset_id": self._dataset._id, "dataset_record_id": record_id, "experiment_id": self._id}
+            output_data = None
+            try:
+                output_data = self._task(input_data)  # FIXME: support config?
```

Comment:
https://docs.datadoghq.com/llm_observability/experiments/#decorators Docs say that `task()` should accept `input_data` of type `Dict[str, Any]` but Dataset record input_data has type of `NonNoneJSONType = Union[str, int, float, bool, List[JSONType], Dict[str, JSONType]]`. Which should it be?